InnerEye/ML/scalar_config.py

```diff
@@ -102,6 +103,17 @@ def scale(labels: List) -> Union[List, Any]:
 
 
 class ScalarModelBase(ModelConfigBase):
+    class_names: List[str] = param.List(class_=str,
+                                        default=[MetricsDict.DEFAULT_HUE_KEY],
+                                        bounds=(1, None),
+                                        doc="The label names for each label class in the dataset and model output "
+                                            "in the case of binary and multi-label classification tasks."
+                                            "The order of the names should match the order of label class indices "
+                                            "in dataset.csv"
+                                            "For multi-label classification, this field is required."
+                                            "For binary classification, this field must be a list of size 1, and "
+                                            "is by default ['Default'], but can optionally be set to a more descriptive "
+                                            "name for the positive class.")
```

Comment:
is it tested that the code works if len(class_names)==1, but it's not the default?